Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperBase.mm

```diff
@@ -582,26 +582,26 @@ NSRange makeNSRange(std::optional<SimpleRange> range)
     // directly in the document DOM, so serialization is problematic. Our solution is
     // to use the root editable element of the selection start as the positional base.
     // That fits with AppKit's idea of an input context.
-    auto selectionRoot = document->frame()->selection().selection().rootEditableElement();
-    auto scope = selectionRoot ? selectionRoot : document->documentElement();
+    RefPtr selectionRoot = document->frame()->selection().selection().rootEditableElement();
+    RefPtr scope = selectionRoot ? selectionRoot : document->documentElement();
     if (!scope)
         return std::nullopt;
 
-    return resolveCharacterRange(makeRangeSelectingNodeContents(*scope), range);
+    return resolveCharacterRange(makeRangeSelectingNodeContents(scope.releaseNonNull()), range);
```

Comment:
releaseNonNull() isn't needed

Reply:
That's true. We should really only `WTFMove()` / `releaseNonNull()` when there is a benefit.